src/decode/xz.rs

```diff
@@ -352,7 +352,7 @@ where
                 )));
             }
             // TODO: properties??
-            lzma2::decode_stream(&mut count_input, output)?;
+            lzma2::Lzma2Decoder::new().decompress(&mut count_input, output)?;
```

Comment:
Nit: now that there is a new struct, it'd be more consistent to import `lzma2::Lzma2Decoder`.